api/checks/summaries/completed.md

```diff
@@ -1,7 +1,23 @@
-Results have successfully been scraped and added to [{{ .HostName }}]({{ .HostURL }}).
+{{ template "_successfully_scraped.md" . }}
 
-They can be compared to the latest `master` run here:
-{{ .DiffURL }}
+There were no regressions detected in the results.
 
-Or, to view _all_ the latest runs for the same revision:
-{{ .SHAURL }}
\ No newline at end of file
+{{ template "_pr_and_master_specs.md" . }}
+
+### Results
+
+Test | `{{ printf "%.7s" .PRRun.FullRevisionHash }}`
+--- | ---
+{{ range $test, $results := .Results -}}
+{{ $test }} | {{ index $results 0 }} / {{ index $results 1 }}
+{{end}}
+{{ if gt .More 0 -}}
+And {{ .More }} others...
+{{ end }}
+
+[Visual comparison of the results]({{ .DiffURL }})
+
+Other views that might be useful:
+- [`{{ printf "%.7s" .PRRun.FullRevisionHash }}` vs `master`@`{{ printf "%.7s" .MasterRun.FullRevisionHash }}`]({{ .DiffURL }})
```

Comment:
This line and "Visual comparison of the results" appear to refer to the same link. Is this repetition intentional?

Reply:
Yeah, in case people aren't quite sure what the visual comparison is.